release-notes/6.0/known-issues.md

```diff
@@ -58,3 +58,46 @@ To fix this, edit your `PATH` environment variable to either remove the `c:\Prog
 The first time SPA apps are run, the authority for the spa proxy might be incorrectly cached which results in the JWT bearer being rejected due to Invalid issuer. The workaround is to just restart the SPA app and the issue will be resolved. If restarting doesn't resolve the problem, another workaround is to specify the authority for your app in Program.cs: `builder.Services.Configure<JwtBearerOptions>("IdentityServerJwtBearer", o => o.Authority = "https://localhost:44416");` where 44416 is the port for the spa proxy.
 
 When using localdb (default when creating projects in VS), the normal database apply migrations error page will not be displayed correctly due to the spa proxy. This will result in errors when going to the fetch data page. Apply the migrations via 'dotnet ef database update' to create the database.
+
+## Windows Desktop (Windows Forms / WPF)
+
+### Issues running applications with Windows Desktop 6.0.2
+
+Some customers are unable to run Windows Desktop (that is, Windows Forms or WPF) applications built against 6.0.2 won't run, if either .NET Windows Desktop runtime 6.0.0 or 6.0.1 are installed, and receieve error messages similar to the following:
+```
+Application: WinFormsApp1.exe
+CoreCLR Version: 6.0.121.56705
+.NET Version: 6.0.1
+Description: The process was terminated due to an unhandled exception.
+Exception Info: System.IO.FileLoadException: Could not load file or assembly 'System.Windows.Forms, Version=6.0.2.0, Culture=neutral, PublicKeyToken=b77a5c561934e089'. The located assembly's manifest definition does not match the assembly reference. (0x80131040)
+File name: 'System.Windows.Forms, Version=6.0.2.0, Culture=neutral, PublicKeyToken=b77a5c561934e089'
+   at WinFormsApp1.Program.Main()   
+```
+
+This is a result of Windows Desktop servicing ref packs in 6.0.2, which was shipped with an incorrect version.
+
+**Fix:**
+* To run Windows Desktop applications built against 6.0.0 or 6.0.1, the Windows Desktop runtime either 6.0.0 or 6.0.1 must be installed. 
```

Comment:
I think an app built against 6.0.0 or 6.0.1 should run correctly if 6.0.2 or higher is installed.

"Built against 6.0.2" isn't really precise either.  It's apps built with the 6.0.102 .NET SDK or higher.

Reply:
> I think an app built against 6.0.0 or 6.0.1 should run correctly if 6.0.2 or higher is installed.

Yes, you're absolutely correct. I just tested it on a brand new VM with only Windows Desktop runtime 6.0.3 installed, and was able to run a Windows Forms app built against 6.0.100.

Reply:
I made updates. How does it look now?
Thank you